lxd/containers_get.go

```diff
@@ -67,7 +67,13 @@ func doContainerGet(d *Daemon, cname string) (*shared.ContainerInfo, Response) {
 	}
 
 	cts, err := c.Render()
-	if err != nil {
+	if err == LxcMonitorStateError {
+		return &shared.ContainerInfo{
+			Name:       cname,
+			Status:     "Error",
+			StatusCode: 112,
```

Comment:
Shouldn't you use the constants above rather than hardcoding the string and code?